release/pkg/filereader/file_reader.go

```diff
@@ -76,15 +83,15 @@ func readShaFile(filename string) (string, error) {
 	return "", errors.Errorf("Error parsing shasum file %s", filename)
 }
 
-func readFile(filename string) (string, error) {
+func ReadFile(filename string) (string, error) {
```

Comment:
Exporting this symbol is offers it up as an API to be used from the package. It does more than read a file, it trims space. Unknowing maintainers could use it and have undesirable side effects. I suggest its refactored to only read the file and you wrap an unexported function around the exported `ReadFile` to prevent the undesirable behavior.

Reply:
Done